mappings/net/minecraft/entity/ai/brain/task/TaskBuilder.mapping

```diff
@@ -57,7 +57,7 @@ CLASS net/minecraft/unmapped/C_rcqaryar net/minecraft/entity/ai/brain/task/TaskB
 			ARG 3 function2
 			ARG 4 function3
 			ARG 5 function4
-		METHOD lift1 lift1 (Lcom/mojang/datafixers/kinds/App;)Ljava/util/function/Function;
+		METHOD lift1 (Lcom/mojang/datafixers/kinds/App;)Ljava/util/function/Function;
```

Comment:
What is this, and why the `1`?
Enigma couldn't find any calls, is it just dead code?

Reply:
going to leave this for another PR when i cover entities, i've set up a reminder on discord